libraries/rush-lib/src/pluginFramework/PhasedCommandHooks.ts

```diff
@@ -143,6 +149,11 @@ export class PhasedCommandHooks {
     [IOperationRunnerContext & IOperationExecutionResult]
   > = new AsyncSeriesHook(['runnerContext'], 'afterExecuteOperation');
 
+  /**
+   * Hook invoked to shutdown long-lived work in plugins.
+   */
+  public readonly shutdown: SyncHook<void> = new SyncHook(undefined, 'shutdown');
```

Comment:
This should be an `AsyncParallelHook<void>` eventually. The invoker will need to find a way to wait for this so that errors can be reflected.